### ISSUE TITLE
DELETE RECOD OF ONE TRANSACTION

### ISSUE THREAD

**Issue by MAPA2023** [moneymanagerex/moneymanagerex#5781]
MMEX version:
 - [ ] 1.6.0
 - [ ] 1.6.1
 - [x] 1.6.2
 - [ ] 1.6.3 Beta
 - [ ] Other (please specify)

Note: bug reporters are expected to have verified the bug still exists
either in the last stable version of MMEX or on updated development code
(master branch).

Operating System:
 - [x] Windows
 - [ ] Mac OSX
 - [ ] Linux 

# Description of the bug

The error is to leave an empty recod in the transaction list. This happens when you delete a transaction. After deleting it goes in the list of deleted transactions. If you cancel from there the problem arises and leaves the trance with no more relation with the various parameters.

# Reproduction

Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:

Expected result:

Actual result:

# Additional information

If you have a backtrace for a crash or a warning, paste it here.

**Comment by tactilis** [comment]
@MAPA2023 This is a duplicate of #5630, where you appear to have already solved the problem.

From MMEX 1.6.4 onwards, the problem won't arise, and any existing orphaned transactions will be automatically deleted.

Could you close this issue please?
